docs/gallery/interactive/pyfar_arithmetics.ipynb

```diff
@@ -0,0 +1,237 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Arithmetic Operations\n",
+    "\n",
+    "Many algorithms require arithmetic operations on signals, such as adding or multiplying two signals, or performing a matrix multiplication between an array and a multi-channel signal. To this end, pyfar implements the operations [add](https://pyfar.readthedocs.io/en/stable/classes/pyfar.audio.html#pyfar.classes.audio.add), [subtract](https://pyfar.readthedocs.io/en/stable/classes/pyfar.audio.html#pyfar.classes.audio.subtract), [multiply](https://pyfar.readthedocs.io/en/stable/classes/pyfar.audio.html#pyfar.classes.audio.multiply), [divide](https://pyfar.readthedocs.io/en/stable/classes/pyfar.audio.html#pyfar.classes.audio.divide), [power](https://pyfar.readthedocs.io/en/stable/classes/pyfar.audio.html#pyfar.classes.audio.power), and [matrix_multiplication](https://pyfar.readthedocs.io/en/stable/classes/pyfar.audio.html#pyfar.classes.audio.matrix_multiplication) in top level functions. The following examples illustrate how to perform arithmetic operations using these functions and corresponding operators `+`, `-`, `*`, `/`, `**`, and `@`. It is best to learn about `LINK-TO-PYFAR-AUDIO-OBJECTS` before continuing with these examples."
```

Comment:
the link can be already inserted.

Reply:
I want to wait with that in case we change names of files or move them to other places in the documentation - that is until the layout of the new homepage is fixed.